html_css/v2/main-course/responsive-css/responsive-css-introduction.md

```diff
@@ -0,0 +1,26 @@
+### Introduction
+"Responsive Design" is an incredibly important concept when it comes to developing for the web. In many ways you are already equipped to create beautiful responsive websites. In this section we're going to fill in the gaps.
+
+### Learning Outcomes
+* You'll learn what 'Responsive Design' is and why it's important.
```

Comment:
Is there a reason we use single quotes here (and in subsequent places), but double quotes in the introduction?

Reply:
Nope.  Should be consistent (double)